model/patch/db.go

```diff
@@ -160,13 +160,15 @@ func ByPatchNameStatusesCommitQueuePaginated(ctx context.Context, opts ByPatchNa
 		match[AliasKey] = evergreen.CommitQueueAlias
 	}
 
-	if !opts.IncludeHidden {
-		match[HiddenKey] = bson.M{"$ne": true}
-	}
 	// This is only used on the user patches page when we want to filter out the commit queue
 	if opts.IncludeCommitQueue != nil && !utility.FromBoolPtr(opts.IncludeCommitQueue) {
 		match[AliasKey] = commitQueueFilter
 	}
+
+	if opts.IncludeHidden != nil && !utility.FromBoolPtr(opts.IncludeHidden) {
```

Comment:
You can just do `if !utility.FromBoolPtr(opts.IncludeHidden)`

Reply:
Ah - in that case you can just use [FromBoolTPtr](https://gist.github.com/hadjri/170a0435ad2d815ceff8426b63f8238d#file-optional-go-L35-L43)